### PR TITLE
Allign wizard with mockups

### PR DESCRIPTION
**Pull request by mareklibra**

### CODE REVIEW

File: src/components/fetching/HostRequirements.tsx [hunk 1]
```diff
@@ -30,7 +43,52 @@ const HostRequirements: React.FC<{
...
+    <>
+      <Button
+        variant={ButtonVariant.link}
+        isDisabled={!hostRequirements}
+        onClick={() => setHostRequirementsOpen(true)}
+        isInline
+        className="host-requirements-link"
+      >
+        <OutlinedQuestionCircleIcon />
+        &nbsp;Minimum hardare requirements
```
Comment (jtomasek) [inline src/components/fetching/HostRequirements.tsx:56]:
```suggestion
        &nbsp;Minimum hardware requirements
```

File: src/components/clusterWizard/ClusterWizardToolbar.tsx [hunk 2]
```diff
@@ -21,7 +29,57 @@ const ClusterWizardToolbar: React.FC<{
   onBack?: () => void;
   onCancel?: () => void;
   onInstall?: () => Promise<void>;
-}> = ({
+};
+
+const ValidationSection: React.FC<ValidationSectionProps> = ({
```
Comment (jtomasek) [inline src/components/clusterWizard/ClusterWizardToolbar.tsx:34]:
```suggestion
const ValidationSectionToggle: React.FC<ValidationSectionProps> = ({
```

**Review by jtomasek — CHANGES_REQUESTED** [review]

File: src/components/ui/DetailList.tsx [hunk 3]
```diff
@@ -27,6 +27,7 @@ export type DetailItemProps = {
     | React.ReactNode;
   idPrefix?: string;
   isHidden?: boolean;
+  classNameValue?: string;
```
[the remark this answers is not in the document]
Comment (mareklibra) [reply src/components/ui/DetailList.tsx:30]: So we can add `className` to the title in the future. We do not have a single shared parent here..

File: src/components/fetching/HostRequirements.tsx (hunk 1 above)
Comment (mareklibra) [reply src/components/fetching/HostRequirements.tsx:56]: done

File: src/components/clusterWizard/ClusterWizardToolbar.tsx (hunk 2 above)
Comment (mareklibra) [reply src/components/clusterWizard/ClusterWizardToolbar.tsx:34]: done

File: src/components/fetching/HostRequirements.tsx [hunk 4]
```diff
@@ -30,7 +43,52 @@ const HostRequirements: React.FC<{
...
+        <Button key="close" variant={ButtonVariant.primary} onClick={onClose}>
+          Close
+        </Button>,
+      ]}
+      onClose={onClose}
+      variant={ModalVariant.medium}
+    >
+      <TextContent>
+        <ContentComponent {...hostRequirements} />
+      </TextContent>
```
[the remark this answers is not in the document]
Comment (mareklibra) [reply src/components/fetching/HostRequirements.tsx]: done

File: src/components/fetching/HostRequirements.tsx [hunk 5]
```diff
@@ -30,7 +49,48 @@ const HostRequirements: React.FC<{
...
+      onClose={onClose}
+      variant={ModalVariant.medium}
+    >
+      <TextContent>
+        {hostRequirements ? (
+          <ContentComponent {...hostRequirements} />
+        ) : (
+          <LoadingState content="Loading hardware requirements ..." />
+        )}
+      </TextContent>
```
Comment (jtomasek) [inline src/components/fetching/HostRequirements.tsx:70]:
```suggestion
      
        {hostRequirements ? (
          <TextContent>
            <ContentComponent {...hostRequirements} />
          </TextContent>
        ) : (
          <LoadingState content="Loading hardware requirements ..." />
        )}
      
```

**Review by jtomasek — APPROVED** [review]
LGTM, just a small nit

**Pull request merged**